core/templates/dev/head/pages/oppia_footer_directive.html

```diff
@@ -25,6 +25,7 @@ <h4 translate="I18N_FOOTER_CONTRIBUTE_ALL_CAPS"></h4>
         <ul>
           <li><a href="https://oppiafoundation.org/volunteer" translate="I18N_FOOTER_GET_INVOLVED"></a></li>
           <li><a href="/teach#teach" translate="I18N_FOOTER_TEACH"></a></li>
+          <li><a ng-if="$ctrl.siteFeedbackFormUrl" ng-href="<[$ctrl.siteFeedbackFormUrl]>" target="_blank" rel="noopener" translate="I18N_SPLASH_SITE_FEEDBACK" ></a></li>
```

Comment:
Move the `ng-if="$ctrl.siteFeedbackFormUrl"` to the `<li>` element so that it is fully hidden if `$ctrl.siteFeedbackFormUrl` is `null`

Reply:
done